lib/ts/recipe/emailpassword/components/features/resetPasswordUsingToken/index.tsx

```diff
@@ -73,17 +73,17 @@ class ResetPasswordUsingToken extends PureComponent<
                       recipeImplementation: this.props.recipe.recipeImpl,
                       config: this.props.recipe.config,
                       onSignInClicked: () => {
-                          void SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(
-                              "signin",
-                              this.props.history
-                          );
+                          void SuperTokens.getInstanceOrThrow().redirectToAuth({
+                              show: "signin",
+                              history: this.props.history,
+                          });
                       },
                       token: this.state.token,
                   };
 
         const enterEmailForm = {
             onBackButtonClicked: () =>
-                SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath("signin", this.props.history),
+                SuperTokens.getInstanceOrThrow().redirectToAuth({ show: "signin", history: this.props.history }),
```

Comment:
need to explicitly set redirectBack to false -> internal use of this function should always explicitly set this value to prevent future bugs.